dynawo/sources/Models/CPP/ModelNetwork/DYNModelStaticVarCompensator.cpp

```diff
@@ -489,18 +489,21 @@ ModelStaticVarCompensator::evalZ(const double& /*t*/) {
 
 void
 ModelStaticVarCompensator::evalG(const double& /*t*/) {
-  g_[0] = (doubleEquals(z_[0], 0.)) ? ROOT_UP : ROOT_DOWN;
-  g_[1] = (doubleEquals(z_[0], 2.)) ? ROOT_UP : ROOT_DOWN;
+  g_[0] = (doubleEquals(z_[modeNum_], 0.)) ? ROOT_UP : ROOT_DOWN;
```

Comment:
I saw you defined an enum to replace num by names for z_ I find it really good would it be possible to do it for g ? (in another commit maybe)

Reply:
Yes it would be in another commit as it seems that the evalG method of the SVC is not correct (see #757 )

Reply:
Good. I don't know if it is like this in all models but it is a good improve